esp/GBPlay/main/tasks/network_manager.c

```diff
@@ -5,97 +5,84 @@
 #include <freertos/event_groups.h>
 #include <freertos/task.h>
 
+#include <esp_event.h>
 #include <esp_log.h>
-#include <esp_wifi.h>
 
 #include "../hardware/wifi.h"
 
-#define TASK_NAME "connection-manager"
+#define TASK_NAME "network-manager"
```

Comment:
Renamed "connection manager" to "network manager" to avoid confusion between a network connection and a socket connection.